models/_sector/nft/trades/MIGRATION.md

```diff
@@ -0,0 +1,156 @@
+# Sector: NFT
+
+We're migrating the NFT sector spells to a new structure which will make implementing and maintaining marketplace abstractions a lot easier and more robust.
+
+## `nft.trades`
+
+- partition_by: `[blockchain, block_date]` 
+- unique_key: `[blockchain, project, project_version, block_number,tx_hash, sub_tx_trade_id]`
+
+| Column Name             | Descriptions                                                       |
```

Comment:
probably not in this markdown file for migration, but we should likely add a readme at the root directory of `_sector/nft/trades` which reuses some of this info, then compliment it with more:
- null / not null column, rather than "if applicable" in description
- accepted values column
- ...maybe more

even though these will be applied in schema file(s), i like the idea of having it laid out in readme for quick pointers

Reply:
agreed. :heavy_check_mark: 
This markdown is mostly just a work document for now, but I think once we've finalized the migration we distill it to a good overview and put it in a readme.